### ISSUE TITLE
Uncaught TypeError: Cannot read property 'buffer' of undefined

### ISSUE THREAD

**Issue by jeffhall-wk** [dannyfritz/atom-readonly#1]
Steps to reproduce:
1. Open command palette
2. Enter text "readonly:toggle"
3. Press "Enter"

**Atom**: 1.22.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [readonly](https://github.com/dannyfritz/atom-readonly) package 0.2.0

### Stack Trace

Uncaught TypeError: Cannot read property 'buffer' of undefined

```
At /Users/jeffhall/.atom/packages/readonly/lib/readonly.js:28

TypeError: Cannot read property 'buffer' of undefined
    at Object.toggle (/packages/readonly/lib/readonly.js:28:56)
    at HTMLElement.readonlyToggle (/packages/readonly/lib/readonly.js:17:41)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:381:36)
    at Object.didConfirmSelection (/Applications/Atom.app/Contents/Resources/app/node_modules/command-palette/lib/command-palette-view.js:144:35)
    at SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app/node_modules/command-palette/node_modules/atom-select-list/src/select-list-view.js:327:26)
    at HTMLDivElement.core:confirm (/Applications/Atom.app/Contents/Resources/app/node_modules/command-palette/node_modules/atom-select-list/src/select-list-view.js:84:20)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:381:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -2:06.6.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
     -1:56 intentions:highlight (input.hidden-input)
 11x -1:55.7.0 readonly:toggle (input.hidden-input)
     -1:18.5.0 tree-view:open-in-new-window (span.name.icon.icon-repo)
     -1:14.4.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
     -1:14.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-outlander-syntax.theme-outlander-ui)
     -0:25.1.0 command-palette:toggle (table.package-keymap-table.table.native-key-bindings.text)
     -0:21.6.0 core:confirm (input.hidden-input)
     -0:21.6.0 readonly:toggle (table.package-keymap-table.table.native-key-bindings.text)
```

### Non-Core Packages

```
apex-adapt-dark-slim-ui 0.4.2 
apex-adapt-dark-ui 0.4.2 
apex-syntax 0.2.0 
atom-toolbar 0.1.6 
busy-signal 1.4.3 
dark-flat-ui 1.2.4 
dart 1.0.1 
file-icons 2.1.14 
flanderized-ui 1.0.1 
intentions 1.1.5 
linter 2.2.0 
linter-ui-default 1.6.10 
minimap 4.29.7 
multi-cursor 2.1.5 
nucleus-dark-ui 0.12.3 
outlander-syntax 2.0.0 
outlander-ui 1.4.0 
read-only-indicator 0.7.0 
readonly 0.2.0 
seti-syntax 1.1.3 
seti-ui 1.9.0 
```

**Comment by dannyfritz** [comment]
This would happen if you don't have a file open. I'm going to investigate Atom Beta 1.23 a little bit because it might make this package obsolete.

**Comment by dannyfritz** [comment]
Should be fixed in v0.3.0. Thanks for the report!
